### PR TITLE
OCLOMRS-1017: Add a workflow badge for the dictionary manager tests

### PR DESCRIPTION
**Pull request by hadijahkyampeire**
[OCLOMRS-1017](https://issues.openmrs.org/browse/OCLOMRS-1017)

- Added a new personalized workflow for the dictionary manager and added its badge to the QA dashboard.

### CODE REVIEW

**Comment by hadijahkyampeire** [comment]
Screenshot 2021-08-03 at 02 21 22

**Comment by sherrif10** [comment]
Thanks @hadijahkyampeire , Does this considers E2E  based work flow ,Do we have configurations for Dictionary manager steps which will handle this dashboard.

**Comment by hadijahkyampeire** [comment]
> Thanks @hadijahkyampeire, Does this considers E2E-based workflow, Do we have configurations for Dictionary manager steps and probably feature file which will handle this dashboard.

@sherrif10 I don't understand what you mean, maybe check this other PR on the OCL repo with the workflow I created, I believe it will capture the status of all cypress tests, though I am not so sure.

File: README.md [hunk 1]
```diff
@@ -33,6 +33,8 @@ ___
 * [![RefApp 3.x Search & Registration](https://github.com/openmrs/openmrs-contrib-qaframework/actions/workflows/refapp-3x-registration.yml/badge.svg)](https://github.com/openmrs/openmrs-contrib-qaframework/actions/workflows/refapp-3x-registration.yml) _In Development_
 ___
 
+### openmrs-ocl-client
+* [![Dictionary Manager](https://github.com/openmrs/openmrs-ocl-client/actions/workflows/dictionary-manager.yml/badge.svg)](https://github.com/openmrs/openmrs-ocl-client/actions/workflows/dictionary-manager.yml)
```
Comment (ibacher) [inline README.md:37]:
```suggestion
* [![RefApp 3.x Search & Registration](https://github.com/openmrs/openmrs-contrib-qaframework/actions/workflows/refapp-3x-registration.yml/badge.svg)](https://github.com/openmrs/openmrs-contrib-qaframework/actions/workflows/refapp-3x-registration.yml) _In Development_

### openmrs-ocl-client
* [![Dictionary Manager](https://github.com/openmrs/openmrs-ocl-client/actions/workflows/dictionary-manager.yml/badge.svg)](https://github.com/openmrs/openmrs-ocl-client/actions/workflows/dictionary-manager.yml)
___
```

**Comment by hadijahkyampeire** [comment]
@sherrif10 @ibacher says yes to the question you asked so I guess we are okay to merge this.

**Review by sherrif10 — APPROVED** [review]
Thanks @hadijahkyampeire  @ibacher  ,LGTM

**Pull request merged**